Gopkg.toml

```diff
@@ -31,27 +31,27 @@
 
 [[constraint]]
   name = "github.com/gravitational/trace"
-  branch = "master"
+  version = "1.1.10"
```

Comment:
I believe you have to put "=1.1.10" if you want specific version. Otherwise next time it can be upgraded to a new patch version.